cadc-pkg-server/src/main/java/org/opencadc/pkg/server/TarWriter.java

```diff
@@ -0,0 +1,239 @@
+*  modify it under the terms of         modifier suivant les termes de
+*  the GNU Affero General Public        la “GNU Affero General Public
+*  License as published by the          License” telle que publiée
+*  Free Software Foundation,            par la Free Software Foundation
+*  either version 3 of the              : soit la version 3 de cette
+*  License, or (at your option)         licence, soit (à votre gré)
+*  any later version.                   toute version ultérieure.
+*
+*  OpenCADC is distributed in the       OpenCADC est distribué
+*  hope that it will be useful,         dans l’espoir qu’il vous
+*  but WITHOUT ANY WARRANTY;            sera utile, mais SANS AUCUNE
+*  without even the implied             GARANTIE : sans même la garantie
+*  warranty of MERCHANTABILITY          implicite de COMMERCIALISABILITÉ
+*  or FITNESS FOR A PARTICULAR          ni d’ADÉQUATION À UN OBJECTIF
+*  PURPOSE.  See the GNU Affero         PARTICULIER. Consultez la Licence
+*  General Public License for           Générale Publique GNU Affero
+*  more details.                        pour plus de détails.
+*
+*  You should have received             Vous devriez avoir reçu une
+*  a copy of the GNU Affero             copie de la Licence Générale
+*  General Public License along         Publique GNU Affero avec
+*  with OpenCADC.  If not, see          OpenCADC ; si ce n’est
+*  <http://www.gnu.org/licenses/>.      pas le cas, consultez :
+*                                       <http://www.gnu.org/licenses/>.
+*
+*  $Revision: 5 $
+*
+************************************************************************
+*/
+
+package org.opencadc.pkg.server;
+
+import ca.nrc.cadc.net.DigestUtil;
+import ca.nrc.cadc.net.HttpDownload;
+import java.io.ByteArrayOutputStream;
+import java.io.FilterOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.io.PrintWriter;
+import java.net.URI;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+import org.apache.commons.compress.archivers.ArchiveEntry;
+import org.apache.commons.compress.archivers.tar.TarArchiveEntry;
+import org.apache.commons.compress.archivers.tar.TarArchiveOutputStream;
+import org.apache.log4j.Logger;
+
+/**
+ *
+ * @author pdowler
+ */
+public class TarWriter {
+    private static final Logger log = Logger.getLogger(TarWriter.class);
+
+    private final TarArchiveOutputStream tout;
+    private final Map<String,List<TarContent>> map = new TreeMap<String,List<TarContent>>();
+    
+    public TarWriter(OutputStream ostream) {
+        this.tout = new TarArchiveOutputStream(ostream);
+        tout.setLongFileMode(TarArchiveOutputStream.LONGFILE_POSIX);
+    }
+    
+    private class TarContent {
+        URL url;
+        String filename;
+        String contentMD5;
+        String emsg;
+    }
+    
+    public void close() throws IOException {
```

Comment:
most of this should be removed; just keep the last 2 lines (flush and close)

Reply:
ok - removed everything that writes the README file out.